blobfuse/src/BlobStreamer.cpp

```diff
@@ -1,6 +1,8 @@
 #include <BlobStreamer.h>
 #include <base64.h>
 
+#define MAX_BLOCK_SIZE_FOR_SINGLE_READ (64 * 1204 * 1024)
```

Comment:
we're opting to not have it be configurable?

Reply:
Block size is still configurable, we are just deciding whether to do download in single thread or in parallel based on 64MB boundary condition.